doc/user/content/serve-results/power-bi.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Power BI"
+description: "How to create dashboards with Power BI"
+aliases:
+  - /third-party/power-bi/
+  - /integrations/power-bi/
+menu:
+  main:
+    parent: "bi-tools"
+    name: "Power BI"
+    weight: 10
+---
+
+You can use [Power BI](https://powerbi.microsoft.com/) to create dashboards
+based on the data maintained in Materialize.
+
+## Database connection details
+
+To set up a connection from Power BI to Materialize, use the native
+[PostgreSQL database driver](https://learn.microsoft.com/en-us/power-query/connectors/postgresql#connect-to-a-postgresql-database-from-power-query-desktop)
+with the following parameters:
+
+Field                  | Value
+---------------------- | ----------------
+Database type          | **PostgreSQL database**
+Server                 | Materialize host name followed by `:6875`
+Database               | **materialize**
+Data Connectivity mode | **DirectQuery**
+Database username      | Materialize user.
+Database password      | App-specific password.
```

Comment:
Should we point users to the docs that explain how to generate an app password?

Reply:
I've followed the structure for the other BI guides that we have like this one here:
> https://materialize.com/docs/serve-results/deepnote/#step-1-create-an-integration

Also, there isn't a dedicated guide on how to generate an app password I believe.